deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -815,6 +815,7 @@ class AccountSetupController: UITableViewController {
             appDelegate.stop()
             appDelegate.close()
             do {
+                //TODO: remove also blobs folder?!
```

Comment:
@r10s @nayooti what do you think about this todo? I think we should clear the blobs folder if the user deleted the account, right?

Reply:
I'll do a seperate PR for this, as it will add some other changes in AccountSetupController that are out of scope of this PR